### PR TITLE
Rs 2022 04 02

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Arcueid/Day 10/10_00_ARC10_3_2.txt [hunk 1]
```diff
@@ -593,12 +629,16 @@ Welcome to our secret banquet hall."
...
-Arcueid lets go of my arm and with a cool, yet joyful smile, has enacted her revenge against me.
+Arcueid lets go of my arm, a cool yet delighted smile on her face.
+ Her revenge is complete.
 }
 [sha:67176ef72eb3814dbf760c24317bac955b71200e]{
 -- Page 211, Offset 24797.
 -- ここが宴会場なら座布団あげたいところだ。
-If this is the banquet hall, then I would want to lay out a sitting cushion.
+// Not sure that the cusiony-ness is the key thing here,
+// assuming banquet hall is simply a 和風 affair
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_3_2.txt:639]: 和風 indeed. She's talking about a 宴会場 like this one where everyone has sitting cushions in rows:
image

Of course I don't really disagree that it's a key thing but there's the context for why it said sitting cushion

File: script/Arcueid/Day 10/10_00_ARC10_3_2.txt [hunk 2]
```diff
@@ -853,13 +902,16 @@ Right as I was forgeting about this mundane world, a pretty lady in a dress pull
 [sha:0361ba830340740e603e3328a6fe13fdaac41b36]{
 -- Page 228, Offset 24848.
 -- 引きこもり、というヤツだろうか。
-So she's one of those hikikomori that I've heard about.
+// Hikikomori would definitely be understood by our readers, but changed it
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_3_2.txt:905]: no complaints from me

**Review by Avuxo — APPROVED** [review]

File: script/Arcueid/Day 10/10_00_ARC10_3_2.txt (hunk 1 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 10/10_00_ARC10_3_2.txt:639]: Yeah, this is what I had in mind. Just want to confirm you are in fact agreeing that the form factor of the chair isn't the important thing, the lack of anywhere to sit is?

Comment (Avuxo) [reply script/Arcueid/Day 10/10_00_ARC10_3_2.txt:634]: correct

**Pull request merged**